src/test/groovy/com/stormpath/tck/logout/LogoutIT.groovy

```diff
@@ -31,46 +28,22 @@ import static org.hamcrest.Matchers.*
 
 @Test
 class LogoutIT extends AbstractIT {
+    private TestAccount account = new TestAccount()
 
-    private final String randomUUID = UUID.randomUUID().toString()
-    private final String accountEmail = "fooemail-" + randomUUID + "@stormpath.com"
-    private final String accountGivenName = "GivenName-" + randomUUID
-    private final String accountSurname = "Surname-" + randomUUID
-    private final String accountPassword = "P@sword123!"
-
-    private final String logoutPath = "/logout"
-    private final String registerPath = "/register"
-    private final String loginPath = "/login"
+    private final String logoutPath = FrameworkConstants.LogoutRoute
```

Comment:
Any reason to do this instead of just referring to (ex.) `FrameworkConstants.LogoutRoute` in code?